program_analyzer.py

```diff
@@ -2,35 +2,34 @@
 # main.py
 #
 # By Benjamin Carter, Dylan Johnson, and Hunter Jenkins
-#  
+#
 # This program constitutes the main of the CoreEngine
 
```

Comment:
TODO (Note to self), Change processFiles() to only process by PR. Rename to process_files() too.